README.md

```diff
@@ -87,6 +87,12 @@ To run the project, execute the following command:
 - Twitter: [@twitterhandle](https://twitter.com/nuga0909)
 - LinkedIn: [LinkedIn](linkedin.com/in/adeshina-adenuga-282036171)
 
+👤 **King Immanuel**
+
+- GitHub: [@Kingno3l]
+- Twitter: [@Kingno3l]
+
```

Comment:
- [ ] @Team great work adding all authors in the readme, however, the handles are supposed to be links to the author's social media accounts eg Twitter LinkedIn, and GitHub, kindly consider make the handles working links